### PR TITLE
[OCONF-781] Doc about templating to include a file

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

File: doc/Advanced.md [hunk 1]
```diff
@@ -1137,6 +1137,34 @@ The following tasks will be merged correctly:
...
+3. Make sure to have single quotes around the URI
+
+Example:
+
+```
+o2-qc --config consul-json://{{ consul_endpoint }}/o2/components/qc/ANY/any/mch-qcmn-epn-full-track-matching --remote -b
+```
+becomes
+```
+o2-qc --config 'apricot://{{ apricot_endpoint }}/o2/components/qc/ANY/any/mch-qcmn-epn-full-track-matching --remote -b
```
Comment (vascobarroso) [inline doc/Advanced.md:1158]: @Barthelemy the single quotes are not closed in this line

Comment (Barthelemy) [reply doc/Advanced.md:1158]: good catch, thanks !

File: doc/Advanced.md [hunk 2]
```diff
@@ -1137,6 +1137,34 @@ The following tasks will be merged correctly:
...
+3. Make sure to have single quotes around the URI
+
+Example:
+
+```
+o2-qc --config consul-json://{{ consul_endpoint }}/o2/components/qc/ANY/any/mch-qcmn-epn-full-track-matching --remote -b
+```
+becomes
+```
+o2-qc --config 'apricot://{{ apricot_endpoint }}/o2/components/qc/ANY/any/mch-qcmn-epn-full-track-matching --remote -b'
```
Comment (knopers8) [inline doc/Advanced.md:1158]:
```suggestion
o2-qc --config 'apricot://{{ apricot_endpoint }}/o2/components/qc/ANY/any/mch-qcmn-epn-full-track-matching' --remote -b
```
but you need them just around the value for `--config`

**Review by knopers8 — APPROVED** [review]
What is there now is correct, but I assume you plan to extend it? It would be good to have a pointer to complete set of instructions that one can use.

**Comment by Barthelemy** [comment]
@knopers8 I don't want to advertise stuff that is not tested/working. Only the include has been tested. We saw the difficulties we had with the use of variables thus I wait a bit. The idea is that `include` is the most important one and can be used already.

**Pull request merged**